go/vt/vtadmin/vtsql/config_test.go

```diff
@@ -98,6 +98,97 @@ func TestConfigParse(t *testing.T) {
 		assert.Equal(t, expectedCreds, cfg.Credentials)
 	})
 
+	t.Run("it uses vtsql-credentials-password to override file password, if passed", func(t *testing.T) {
```

Comment:
extremely minor style thing: this is written more like an rspec/jest test case, whereas go tends to have much shorter names (for context, the full name of this test is `TestConfigParse/it_uses_vtsql-credentials-password_to_override_file_password,_if_passed`, where [this test](https://github.com/vitessio/vitess/blob/main/go/vt/vtadmin/vtctldclient/config_test.go#L49) is just `TestParse/no_credentials_provided`)